components/access-management/kfam/bindings.go

```diff
@@ -77,6 +77,14 @@ func getBindingName(binding *Binding) (string, error) {
 }
 
 func getAuthorizationPolicy(binding *Binding, userIdHeader string, userIdPrefix string) istioSecurity.AuthorizationPolicy {
+	istioIGWPrincipal := GetEnvDefault(
+		"ISTIO_INGRESS_GATEWAY_PRINCIPAL",
+		"cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account")
```

Comment:
These two variables seem like it's trying to point to resources(service accounts) on the cluster. Are these resources something we already have? Or are they something that will get created when we update the base manifest?

Just making sure of this so that we don't get unexpected errors.

Reply:
This is a new thing so we would not have it in our current cluster, I have added notes to https://jirab.statcan.ca/browse/BTIS-657 to remind us to add the new ref, or at least make sure it is indeed added.